SharedPackages/BrowserServicesKit/Sources/BrowserServicesKit/ContentScopeScript/ContentScopeUserScript.swift

```diff
@@ -222,7 +222,12 @@ extension ContentScopeUserScript: WKScriptMessageHandlerWithReply {
         propagateDebugFlag(message)
         // Don't propagate the message for ContentScopeScript non isolated context
         if message.name == MessageName.contentScopeScripts.rawValue {
-            return (nil, nil)
+            guard let dict = message.messageBody as? [String: Any],
```

Comment:
@afterxleep @shakyShane Are there any security implications to allowing DuckPlayerNative messages to propagate?